t/sanity.t

```diff
@@ -7,7 +7,7 @@ use Test::Nginx::Socket::Lua;
 #workers(2);
 #log_level('warn');
 
-repeat_each(2);
+repeat_each(1);
```

Comment:
Oh, sorry, because of testing function add_server(), add_peer(), remove_peer(), remove_server(), the same upstream take the server is only one,If  run twice the different response results every time. I just want to call once, so i change.
 I'm not sure if it's a good idea, please give me some tips and I'm glad to learn from your idea.:D